Sources/VersionChecker/VersionChecker.swift

```diff
@@ -58,12 +60,11 @@ public struct VersionChecker {
 
     private func isAppOutdated(version: Version) -> Bool {
         guard let installedVersionTag = VersionUtils.getCurrentlyInstalledVersion()?.tag,
-              let latestProductionVersion = version.getLatestPublishedVersion(for: .production),
-              version.minVersion.compare(latestProductionVersion.tag, options: .numeric) == .orderedAscending
```

Comment:
As discussed yesterday, the `version.minVersion.compare(latestProductionVersion.tag, options: .numeric) == .orderedAscending` is still missing